additional/fix.convert.npz.py

```diff
@@ -0,0 +1,64 @@
+# This script reformats samples to make them backwards compatible between WisecondorX versions.
+# It can also be used to transform .npz files from the original WISECONDOR to WisecondorX.
+# Note that is only compatible with .npz files resulting from the convert version, other
+# functions (newref, predict) should be re-run.
+
+# Usage: python fix.convert.npz.py input.npz output.npz
+
+import sys
+import numpy as np
+
+_, in_npz, out_npz = sys.argv
+
+def toolConvert():
```

Comment:
What are you trying to do here?

Reply:
.npz files contain references to functions when they are badly generated. This function needs to be present to enable conversion of original WISECONDOR files

Reply:
So the return value is `1` or does the `1` return as `True`?

Reply:
It returns nothing -- but it doesn't really matter: something in WISECONDOR's .npz files calls this function, but its result is not used.

Reply:
The 1 is there for syntax reasons